resources/views/layouts/app.blade.php

```diff
@@ -69,8 +70,9 @@
                             </li>
                         @endguest
                     </ul>
-                    @if (Route::has('register'))
-                        <a href="{{ route('register') }}">Register</a>
+                    
+                    @if (Auth::check())
```

Comment:
Still using `Auth::check()`. It's nicer if we just use the blade directive `@auth` instead